pkg/kritis/admission/admission.go

```diff
@@ -230,6 +233,18 @@ func createDeniedResponse(ar *v1beta1.AdmissionReview, message string) {
 func reviewImages(images []string, ns string, pod *v1.Pod, ar *v1beta1.AdmissionReview, config *Config) {
 	// NOTE: pod may be nil if we are reviewing images for a replica set.
 	glog.Infof("Reviewing images for %s in namespace %s: %s", pod, ns, images)
+	client, err := admissionConfig.fetchMetadataClient(config)
+	if err != nil {
+		errMsg := fmt.Sprintf("error getting metadata client: %v", err)
+		glog.Errorf(errMsg)
+		createDeniedResponse(ar, errMsg)
```

Comment:
It's clearer if function output is a returned parameter from a function - it looks like ar may only be return, could it be moved over?

i.e.
func reviewImages(...) *v1beta1.AdmissionReview

Reply:
That's a good point. Given the style of passing this as pointer receivers, I'll keep this as is for now, and consider a larger refactoring later.